api/cron-jobs/projects.js

```diff
@@ -1,332 +1,305 @@
-require('dotenv').config();
-const airTable = require('../helpers/airTable');
-const arraysHelpers = require('../helpers/arrays');
-const axios = require('axios').default;
-const urlsHelpers = require('../helpers/urls');
-const projectsHelpers = require('../helpers/projects');
-const videosService = require('../services/videos');
-const logging = require('../services/logging');
-const api_key = process.env.JIRA_API_KEY;
-const email = process.env.JIRA_EMAIL;
-const resourcingJiraBoardName = 'RES';
-const recruiterAssignedJiraColumnName = 'Recruiter Assigned';
-const projectJiraBoardName = 'IT';
-const volunteerSearch = 'Volunteer Search';
-const volunteerIntroduction = 'Volunteer Introduction';
-const activityUnderway = 'Activity Underway';
+require('dotenv').config()
```

Comment:
Same point as before that I think this can be updated to an `import` (please see earlier comment)